settings.gradle.kts

```diff
@@ -32,4 +32,3 @@ include(":fuel-kotlinx-serialization")
 include(":fuel-moshi")
 include(":fuel-test")
 include(":fuel-json")
-include(":sample")
```

Comment:
[Q] By doing this it means that it won't recognize as a module in IntelliJ? Is this the intended effect?

Reply:
I don't think so

Reply:
Intellij IDEA able to search everything under new source set. It should detect sample.